wiki/Tournaments/Mappool/en.md

```diff
@@ -0,0 +1,95 @@
+  - forcemod
+  - forcedmod
+  - FM
+  - mixed mod
+  - MM
+  - tiebreaker
+---
+
+# Mappool
+
+A **mappool** is a collection of [beatmaps](/wiki/Beatmap) used in [tournament](/wiki/Tournaments) matches. These mappols may be divided into sections based on which [mods](/wiki/Gameplay/Game_modifier) are used, in which case they are sometimes called *modpools*, or by type of beatmap. The individual beatmaps are said to occupy a numbered *mappool slot* within each section of the mappool. People who create mappools are called *mappoolers* or *mappool selectors*, and the process itself is called *mappooling*.
+
+## Mappool sections
+
+Tournaments except osu!mania ones typically include *Hidden*, *Hard Rock*, and *Double Time* sections of the mappool, which as their names suggest require the respective mods to be active. osu!mania tournaments instead use *Rice*, *LN*, *Hybrid*, and *SV*.
+
+### No Mod
+
+**No Mod** (***NM***) refers to not using any mods. Many tournaments require some mods to be used by default as part of their rules or format, such as [No Fail](/wiki/Game_modifier/No_Fail) or [ScoreV2](/wiki/Game_modifier/ScoreV2), which become exceptions to this notion.
+
+### Free Mod
+
+**Free Mod** (***FM***) refers to being free to choose any mod or mod combination. Some tournaments also provide rules that specify additional criteria such as what mods are allowed and in what combinations, or whether having no mods is allowed when Free Mod is specified.
+
+### Force Mod
+
+**Force Mod** (***FM***) is otherwise the same as [Free Mod](#free-mod), except that it specifies that using at least one mod is mandatory.
+
+### Mixed Mod
+
+**Mixed Mod** (***MM***) means each player in the team has to pick one unique mod or mod combination each. The specific requirements may vary between tournaments.
+
+### Rice
+
+**Rice** is an osu!mania community term for regular notes, in reference to regular notes looking like rice. Rice maps thus contain a lot of regular notes.
+
+### LN
+
+**LN** in osu!mania stands for long note, and thus LN maps contain a lot of them.
+
+### Hybrid
+
+**Hybrid** beatmaps contain both regular and long notes, i.e. is a category in between Rice and LN.
+
+### SV
+
+**SV** stands for slider velocity, which in osu!mania affects scroll speed. SV beatmaps make use of scroll speed variation gimmicks.
+
+### Tiebreaker
+
+**Tiebreakers** are beatmaps used to break ties. These typically focus on a variety of skills, consistent performance, and are usually longer than other such that the better player overall wins. The mod requirements vary between tournaments.
+
+## Mappool slots
+
+Tournament mappools are usually structured such that mappool slots correspond to specific types of beatmaps that require different skills to perform well on them. This encourages strategy and coordination when players and teams pick beatmaps in matches.
+
+In the [osu! game mode](/wiki/Game_mode/osu!), the the beatmap types for each slot number are standardised:
```

Comment:
an
```suggestion
In the [osu! game mode](/wiki/Game_mode/osu!), the beatmap types for each slot number are standardised:
```

Reply:
eh i wouldn't call this "standardized" orz

while it's true that \*most\* tournament mappools do follow this format, to my knowledge there has been any convention formalising that "this slot should be exclusively reserved for this sort of map" and so on

Reply:
imo a better sentence for this would be

```suggestion
In the [osu! game mode](/wiki/Game_mode/osu!), the beatmap types for each slot number typically follow the subsequent skillset arrangement:
```

Reply:
yes that was the intention and is probably good to clarify